cmd/kubeadm/app/cmd/reset.go

```diff
@@ -49,6 +49,11 @@ var (
 		The reset process does not clean your kubeconfig files and you must remove them manually.
 		Please, check the contents of the $HOME/.kube/config file.
 	`)
+
+	cniCleanupInstructions = dedent.Dedent(`
+		The reset process does not clean cni configuration nor cni-related binaries.
+		To do so, you must remove /etc/cni/net.d and /opt/cni/bin
```

Comment:
i think we should exclude `/opt/cni/bin` here as it's managed by our packages - `kubernetes-cni` right now and `kubelet` in the future.